CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Nakadi
+
+**Thank you for your interest in Nakadi. Your contributions are highly welcome.**
+
+There are multiple ways of getting involved:
+
+- [Report a bug](#report-a-bug) 
+- [Suggest a feature] (#suggest-a-feature) 
+- [Contribute code] (#contribute-code) 
+
+Below are a few guidelines we would like you to follow.
+If you need help, please reach out to us by opening an issue.
+
+## Report a bug 
+Reporting bugs is one of the best ways to contribute. Before creating a bug report, please check that an [issue] (https://github.com/zalando/nakadi/issues) reporting the same problem does not already exist. If there is an such an issue, you may add your information as a comment.
```

Comment:
fyi: space in between word and link brackets is stopping markdown from creating a proper link ...